1041/en.md

```diff
@@ -16,13 +19,6 @@ To solve this problem we need to know if all the cities are directly or indirect
 
 For my convenience I will implement this in a different way.
```

Comment:
Remove this line as it became irrelevant due to prior removal.

Reply:
solved